DEBUG:root:received {"dest":"n5","body":{"type":"init","node_id":"n5","node_ids":["n1","n2","n3","n4","n5"],"msg_id":1},"src":"c4","id":4}
INFO:root:node n5 initialized
DEBUG:root:sending {"dest": "c4", "src": "n5", "body": {"msg_id": 1, "in_reply_to": 1, "type": "init_ok"}}
/home/peter/Documents/tf/lab3/raft.py:114: DeprecationWarning: randrange() will raise TypeError in the future
  sec = random.randrange(MIN_HB,MAX_HB)
ERROR:root:fatal exception in handler
Traceback (most recent call last):
  File "/usr/lib/python3.10/random.py", line 303, in randrange
    istart = _index(start)
TypeError: 'float' object cannot be interpreted as an integer

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/peter/Documents/tf/lab3/ms.py", line 27, in exitOnError
    fn(*args)
  File "/home/peter/Documents/tf/lab3/raft.py", line 198, in handle
    leader_alive_checker()
  File "/home/peter/Documents/tf/lab3/raft.py", line 114, in leader_alive_checker
    sec = random.randrange(MIN_HB,MAX_HB)
  File "/usr/lib/python3.10/random.py", line 309, in randrange
    raise ValueError("non-integer arg 1 for randrange()")
ValueError: non-integer arg 1 for randrange()
